src/parameterizations/vertical/MOM_internal_tide_input.F90

```diff
@@ -114,11 +114,16 @@ subroutine set_int_tide_input(u, v, h, tv, fluxes, itide, dt, G, GV, US, CS)
                         ! equation of state.
   logical :: avg_enabled  ! for testing internal tides (BDM)
   type(time_type) :: time_end        !< For use in testing internal tides (BDM)
+  real :: HZ2_T3_to_W_m2  ! unit conversion factor for TKE from internal to mks
```

Comment:
As written, `HZ2_T_3_to_W_m3` has units of `[m3 T3 H-1 Z-2 s-3 ~> 1 or m3 kg-1]`, which should be documented, but it does not convert anything to W m-2, so the name is a bit misleading.  Similar issues apply to `W_m2_to_HZ2_T3`.

Reply:
The code is looking correct now, but please add unit descriptions in the comments for `HZ2_T3_to_W_m2` and `W_m2_to_HZ2_T3`.